src/main/java/com/osiris/autoplug/client/tasks/updater/search/CustomCheckURL.java

```diff
@@ -39,7 +39,11 @@ public SearchResult doCustomCheck(String url, String currentVersion) {
                 release = Json.getAsJsonArray(url)
                         .get(0).getAsJsonObject();
             } catch (Exception e) {
-                throw e;
+                try {
+                    release = Json.getAsJsonObject(url);
+                } catch (Exeption ex){
+                    throw ex;
+                }
```

Comment:
you should be able to check the jsonElement to see if its a object or array further above before accessing it.
What about the key traversal thing btw?

Reply:
it now checks if its an object or array
i have looked but am failing to find a way to traverse the keys that i understand well enough to try to implement

Reply:
done